frontend/src/pages/devices/Detail.tsx

```diff
@@ -205,7 +205,7 @@ export const Detail = () => {
         return a.endAt.localeCompare(b.endAt)
       }
 
-      return b.startAt.localeCompare(a.startAt)
+      return a.startAt.localeCompare(b.startAt)
```

Comment:
Sorry, I guess I didn't realize that this one was currently `b` `a`.... it's the the completed section that needs to change, not the others.